recipes/pgx-variant-tools/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pgx-variant-tools" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 24c4fd0a5478dadcd4a837a9823c329d153035dbba994b141f129b2e629b2a2d
+
+build:
+  number: 0
+  entry_points:
+    - vcf2sequence = variant_tools.cli:vcf2sequence
+  script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - biopython
+    - edlib
+    - pyinterval
+    - requests
+    - click
+    - cyvcf2
+
+test:
+  imports:
+    - variant_tools
+  commands:
+    - vcf2sequence --help
+
+about:
+  home: https://github.com/LUMC/pgx-variant-tools
+  license: MIT License
```

Comment:
Should be `MIT` not `MIT License`

Reply:
@dpryan79 @rhpvorderman Btw, I tried to "fix" this upstream in the pypi package where I used `conda skeleton` from. Now I can't upload to pypi anymore because `MIT` is not a valid choice there.

Reply:
You can just mention the license file in pypi. Also the names won't match between various sources, that's normal.